inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/rocks/inspectit/oce/core/config/InspectitConfig.java

```diff
@@ -0,0 +1,9 @@
+package rocks.inspectit.oce.core.rocks.inspectit.oce.core.config;
+
+import lombok.Data;
+
+@Data
+public class InspectitConfig {
+
+    private String serviceName;
```

Comment:
It is very important that we carefully document this class and corresponding configuration file as well